src/collective/lazysizes/upgrades/v2/configure.zcml

```diff
@@ -25,6 +25,13 @@
         handler="..cook_javascript_resources"
         />
 
+    <genericsetup:upgradeDepends
```

Comment:
This is the way to fix this problem: add ``import_profile`` to the ``upgradeDepends`` directive
https://github.com/zopefoundation/Products.GenericSetup/issues/7#issuecomment-237272961